src/IdentityServer/Configuration/DependencyInjection/Options/LoggingOptions.cs

```diff
@@ -22,7 +22,6 @@ public class LoggingOptions
     public ICollection<string> BackchannelAuthenticationRequestSensitiveValuesFilter { get; set; } =
         new HashSet<string>
         {
-            // TODO: IdentityModel
```

Comment:
Review: Unclear what this TODO was for.

Reply:
Likely we were waiting on a constant to be defined and released in a new version of IdentityModel.